assets/components/contribute/contribute.scss

```diff
@@ -52,6 +52,13 @@
       display: none;
     }
   }
+
+    &:before {
+      @include multiline(4, gu-colour(garnett-neutral-4));
+      background-color: #fff;
+      display: block;
+      content: '';
```

Comment:
Just to double check, are we changing the content of the HTML via CSS? Probably that is ambiguous in the future.

Reply:
You need to specify something for the `content` of the `before` and `after` pseudo-elements otherwise they [don't appear](https://stackoverflow.com/questions/9599811/why-do-i-need-an-empty-content-property-on-an-after-pseudo-element) in the flow.